examples/Python/src/lib/WebServer.lf

```diff
@@ -0,0 +1,84 @@
+/**
+ * @file
+ * @author Shulu Li
+ * @brief Reactor for handling HTTP requests in Python.
+ */
+target Python {
+  keepalive: true
+}
+
+/**
+ * @brief A reactor that starts a FastAPI server to handle HTTP requests.
+ *
+ * The `port` parameter is the port number on which the server listens for requests.
+ *
+ * The `endpoint` parameter is the endpoint at which the server listens for requests.
+ *
+ * The `request` output is a tuple of two values: the request ID and the request data.
+ *
+ * The `response` input is a tuple of two values: the request ID and the response data. Request ID
+ * is required to respond to the correct request. Use a physical connection to connect the
+ * `response` input to avoid STP violations.
```

Comment:
It's a little odd to read about STP violations in the docs of a reactor, which could easily be used in unfederated programs as well. Why are any of these examples federations at all? At a quick glance, it isn't clear to me. Do they have to be?

Reply:
For the minimal +1 examples, they can be federated or unfederated and it wouldn't have much effect on the behavior of the program. I will change these examples to unfederated execution. However for the distributed logging example, we want to show the consistency-availability tradeoff that could be achieved with federated execution using `STA`.